build.gradle

```diff
@@ -217,8 +217,13 @@ dependencies {
   functionalTestCompile group: 'org.assertj', name: 'assertj-core', version: '3.14.0'
 
   smokeTestCompile sourceSets.main.runtimeClasspath
+  smokeTestCompile sourceSets.test.runtimeClasspath
```

Comment:
This should not be included. Alter tests to avoid whole lot dependencies from test sources